packages/@aws-cdk/aws-codepipeline-actions/README.md

```diff
@@ -324,9 +324,69 @@ This package defines the following actions:
   changes from the people (or system) applying the changes.
 * **CloudFormationExecuteChangeSetAction** - Execute a change set prepared previously.
 
+##### Lambda deployed through CodePipeline
```

Comment:
This section makes me feel that maybe we should merge the README files of aws-codepipeline and aws-codepipeline-actions. It feels artificial to have to jump between the two... What do you think?

Reply:
I don't have any philosophical objections to that, I do have one practical concern though: nesting. As you can see, we've reached H5 in the ReadMe for `codepipeline-actions`, and I assume we'll need to increase the nesting of the headings by at least one level if we move them to the `codepipeline` ReadMe.